packages/tpetra/core/example/Finite-Element-Assembly/fem_assembly_InsertGlobalIndices_DP.hpp

```diff
@@ -200,9 +202,12 @@ int executeInsertGlobalIndicesDP(const comm_ptr_t& comm, const struct CmdLineOpt
   RCP<TimeMonitor> timerElementLoopMatrix = rcp(new TimeMonitor(*TimeMonitor::getNewTimer("3) ElementLoop  (Matrix)")));
```

Comment:
I've expressed my feelings about `RCP<TimeMonitor>` before.  No worries for now, though :-) .